logi/logi.go

```diff
@@ -12,35 +12,35 @@ type WLogger interface {
 
 // Logger represents common interface for logging functionality
 type Logger interface {
-	// WithContext will return logger with filled basic fields (email, company, corelationID)
-	WithContext(ctx context.Context) Logger
+	// With returns logger with basic fields based on context and your custom fields
+	With(ctx context.Context, keysAndValues ...any) Logger
 
-	// Errorf logs a templated message with the provided error
-	Errorf(format string, err interface{}, args ...interface{})
+	// Err is a shorter version of With(ctx, "error", err)
+	Err(err error) Logger
 
-	// Errorw logs a message with optional fields
-	Errorw(msg string, err interface{}, keysAndValues ...interface{})
+	// Errorf logs a templated message with the provided error
```

Comment:
Can I do:
```go
Log().With(ctx, { field: value }).Error("Something went wrong but I don't have an error variable")
```
If so, then maybe I would remove the "with the provided error" part of the comment, since its not necessary to actually provide an error

Reply:
True. Also it would be 
```
Log().With(ctx, field, value).Error("Something went wrong but I don't have an error variable")
```